### ISSUE TITLE
editorial blockquote font size

### ISSUE THREAD

**Issue by notlee** [Financial-Times/o-quote#67]
The editorial block quote uses `o-editorial-typography` to match the size of article body copy,
18px increasing to 20px on larger viewports.

ft.com article: https://www.ft.com/kitchenus-sinkus-articlus
<img width="733" alt="Screenshot 2020-07-07 at 12 16 35" src="https://user-images.githubusercontent.com/10405691/86772436-f1cc4e00-c04b-11ea-9446-29996bd3eea9.png">

However live blogs body does not increase to 20px, its always 18px. This means the editorial 
blockquote looks too large.

live blogs: https://www.ft.com/content/44544ce1-b225-3be0-8bbb-013a5824cf5e
<img width="722" alt="Screenshot 2020-07-07 at 12 16 58" src="https://user-images.githubusercontent.com/10405691/86772568-2e984500-c04c-11ea-9498-c31525b454d5.png">

### Suggested Solutions

Work in progress. Notes:
- both live blogs and the article page use o-quote via n-content-blockquote. We can't easily change markup and therefore may have to solve this in the short term by inheriting font size / line height for the editorial quote and applying the increased font size for the article page [with overrides](https://github.com/Financial-Times/next-article/blob/0a4aa4222162b935addebd2477bec408f34ac77c/client/components/article/_main.scss#L87)

**Comment by JakeChampion** [comment]
I presume Live Blogs has stated somewhere that they don't want a larger font size for quotes?

**Comment by notlee** [comment]
yeah, this was recently released as [part of this work](https://github.com/Financial-Times/n-content-body/pull/127). Before then the [non-editorial block quote](https://registry.origami.ft.com/components/o-quote@4.1.4#demo-standard) style is 18px Metric

**Comment by JakeChampion** [comment]
I can't see where Live Blogs has said it wants 18px, is it in a sketch file somewhere?

**Comment by notlee** [comment]
Josh raised this in a direct message on Slack. The original intention was to match the font size, I didn't realise live blogs differs from article pages by design
